okhttp/src/commonMain/kotlin/okhttp3/internal/-UtilCommon.kt

```diff
@@ -299,15 +299,16 @@ fun Closeable.closeQuietly() {
  * @param file a file in the directory to check. This file shouldn't already exist!
  */
 internal fun FileSystem.isCivilized(file: Path): Boolean {
-  sink(file).use {
-    try {
-      delete(file)
-      return true
```

Comment:
@swankjesse  This use seems broken, specifically the `return true` escapes the close in `use`.

Reply:
upstream fix https://github.com/square/okio/pull/1188

Reply:
If we fix upstream in Okio, are we done?

Reply:
I don't think so, we can hold off on this until an okio release if it's soon.  But I think we need the other fixes also.

Reply:
I'll copy the use function from okio into here and remove once released.